extensions/amp-inline-gallery/1.0/component.type.js

```diff
@@ -23,7 +23,7 @@ InlineGalleryDef.Props;
  *   children: !Array<PreactDef.Renderable>
  * }}
  */
-InlineGalleryDef.PaginationProps;
+BentoInlineGalleryDef.BentoPaginationProps;
```

Comment:
WDYT, is the second `Bento` prefix needed here? 
`BentoInlineGalleryDef.PaginationProps` seems sufficiently clear.

Reply:
updated